_opensearch/install/windows.md

```diff
@@ -0,0 +1,249 @@
+
+The following recommended settings will allow you to:
+
+- Bind OpenSearch to an IP or network interface on the host.
+- Set initial and max JVM heap sizes.
+- Define an environment variable that points to the bundled JDK.
+
+If you ran the security demo script, then you will need to manually reconfigure settings that were modified. Refer to [Security configuration]({{site.url}}{{site.baseurl}}/opensearch/configuration/) for guidance before proceeding.
+{:.note}
+
+Before modifying any configuration files, it's always a good idea to save a backup copy before making changes. The backup file can be used to revert any issues caused by a bad configuration.
+{:.tip}
+
+1. Open the `opensearch-{{site.opensearch_version}}\config` folder.
+1. Open the `opensearch.yml` file with a text editor.
+1. Add the following lines.
+   ```bash
+   # Bind OpenSearch to the correct network interface. Use 0.0.0.0
+   # to include all available interfaces or specify an IP address
+   # assigned to a specific interface.
+   network.host: 0.0.0.0
+
+   # Unless you have already configured a cluster, you should set
+   # discovery.type to single-node, or the bootstrap checks will
+   # fail when you try to start the service.
+   discovery.type: single-node
+
+   # If you previously disabled the security plugin in opensearch.yml,
+   # be sure to re-enable it. Otherwise you can skip this setting.
+   plugins.security.disabled: false
+   ```
+1. Save your changes and close the file.
+1. Specify initial and max JVM heap sizes.
+   1.  Open the `opensearch-{{site.opensearch_version}}\config` folder.
+   1.  Open the `jvm.options` file with a text editor.
+   1. Modify the values for initial and max heap sizes. As a starting point, you should set these values to half of the available system memory. For dedicated hosts this value can be increased based on your workflow requirements.<br>
+    As an example, if the host machine has 8 GB of memory, then you might want to set the initial and maximum heap sizes to 4 GB:
+    ```bash
+    -Xms4g
+    -Xmx4g
+    ```
+   1. Save your changes and close the file.
+1. Specify the location of the included JDK. 
+    1. In the search box next to **Start** on the taskbar, enter `edit environment variables for your account` or `edit the system environment variables`. To edit the system environment variables, you need admin rights. User environment variables take precedence over system environment variables.
+    1. Select **Edit environment variables for your account** or **Edit the system environment variables**. 
+    1. If the **System Properties** dialog comes up, in the **Advanced** tab, select **Environment Variables**.
```

Comment:
If the **System Properties** dialog opens, got to the **Advanced** tab and select **Environment Variables**.
Read better?

Reply:
The thing is - it opens already on the Advanced tab, so all you do is select EV. Will reword.